wealthy-senior-execs/config/_colour.scss

```diff
@@ -0,0 +1,17 @@
+// Red shades
```

Comment:
Do these need a prefix? I'm just wondering if there might be some confusion when you declare $blue but it ends up using the AP blue. Or is the intention to only import the individual files? If that is the case do we want an oleg-brand-kit/wealthy-senior-execs/_config.scss file that imports the three files?

Reply:
Hmm good question... for styles on web.agep we're using the same classes as AP modules (.ap-c-btn, for example). I think it's fine as is?

Good shout about the _config.scss file!